src/firebase/models/rooms.js

```diff
@@ -0,0 +1,43 @@
+import {
+  collection,
+  doc,
+  onSnapshot,
+  setDoc,
+  updateDoc,
+  getDoc
+} from "firebase/firestore";
+import { db } from "../firestore";
+
+export const roomsCollection = collection(db, "rooms");
+
+export function onRoomUpdate(roomID, onUpdate) {
+  const roomDocument = doc(roomsCollection, roomID);
+
+  onSnapshot(roomDocument, snapshot => {
+    onUpdate(snapshot.data());
+  });
+}
```

Comment:
A function that allows us to update room info in other components when changes happen